content/v1.11-docs/reference/cmctl.md

```diff
@@ -226,9 +226,9 @@ of these commands are subject to change or removal in future releases.
 Sub-commands are available to create different resources:
 
 ##### CertificateSigningRequest
-To create a [CertificateSigningRequest](./kube-csr.md), use
+To create a [CertificateSigningRequest](../usage/kube-csr.md), use
 ```console
-cmctl x create csr`
+cmctl x create csr
```

Comment:
note: these were fixed in `docs` - this is just making the same changes for `v1.11-docs`